sdk/requests/__tests__/getUserInfo.test.js

```diff
@@ -0,0 +1,100 @@
+import { fetch } from 'react-native-ssl-pinning';
+import { userInfoEndpoint } from '../../utils/endpoints';
+import getUserInfo from '../getUserInfo';
+import { getParameters } from '../../configuration';
+
+jest.mock('../../configuration');
+
+jest.mock('react-native-ssl-pinning', () => ({
+  fetch: jest.fn(),
+}));
+
+describe('getUserInfo', () => {
+  it('calls getUserInfo with correct accessToken', async () => {
+    const code = 'f24df0c4fcb142328b843d49753946af';
+    const redirectUri = 'uri';
+    getParameters.mockReturnValue({
+      clientId: '898562',
+      clientSecret: 'cdc04f19ac2s2f5h8f6we6d42b37e85a63f1w2e5f6sd8a4484b6b94b',
+      accessToken: 'c9747e3173544b7b870d48aeafa0f661',
+      redirectUri,
+      code,
+    });
+
+    fetch.mockImplementation(() =>
+      Promise.resolve({
+        status: 200,
+        json: () =>
+          Promise.resolve({
+            nombre_completo: 'test',
+            primer_apellido: 'test',
+            primer_nombre: 'testNombre',
+            segundo_apellido: 'testApellido',
+            segundo_nombre: 'testSegundoNombre',
+            sub: '5968',
+            uid: 'uy-cid-12345678',
+          }),
+      }),
+    );
+    const response = await getUserInfo();
+
+    expect(fetch).toHaveBeenCalledWith(userInfoEndpoint, {
+      method: 'GET',
+      sslPinning: {
+        certs: ['certificate'],
+      },
+      headers: {
+        Authorization: `Bearer c9747e3173544b7b870d48aeafa0f661`,
+        'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
+        Accept: 'application/json',
+      },
+    });
+    expect(response).toStrictEqual({
+      nombre_completo: 'test',
+      primer_apellido: 'test',
+      primer_nombre: 'testNombre',
+      segundo_apellido: 'testApellido',
+      segundo_nombre: 'testSegundoNombre',
+      sub: '5968',
+      uid: 'uy-cid-12345678',
+    });
+  });
+
+  it('calls getUserInfo with incorrect clientId or clientSecret or returns incorrect code', async () => {
```

Comment:
```suggestion
  it('calls getUserInfo with incorrect clientId or clientSecret or returns incorrect code', async () => {
  getParameters.mockReturnValue({
      clientId: '',
      clientSecret: '',
      code: 'incorrectCode',
    });
```
Si no agregas eso, el jest se queda con los valores mockeados en el test anterior y aunque devuelve lo que queremos, no esta testeando de verdad lo que queremos. Este error lo acabo de ver también en el archivo `getToken.test.js` y `refreshToken.test.js` en el mismo it, si te animas a cambiarlo, un éxito. De paso, si cambias el archivo `refershToken.test.js` sácale los comentarios que tiene que sino es la única función de test que tiene comentarios 😅